data/wanderer/wanderers start.txt

```diff
@@ -1030,14 +1030,14 @@ mission "Wanderers: Alpha Surveillance C (Pug)"
 				`	(Yes.)`
 				`	(No.)`
 					decline
-			`	You try to get several of the Pug to talk to you, but most of them pointedly ignore you. Eventually, probably because you've made it clear that you are not going away, one of them approaches you carrying a small black box. It speaks to the box in its language, and the box says, "Hello you are a human you are from planet earth are you not." It seems that the box is a rudimentary translation device.`
+			`	You try to get several of the Pug to talk to you, but most of them pointedly ignore you. Eventually, probably because you've made it clear that you are not going away, one of them approaches you carrying a small black box. It speaks to the box in its language, and the box says, "Hello you are a human. You are from planet earth are you not?" It seems that the box is a rudimentary translation device.`
```

Comment:
```suggestion
			`	You try to get several of the Pug to talk to you, but most of them pointedly ignore you. Eventually, probably because you've made it clear that you are not going away, one of them approaches you carrying a small black box. It speaks to the box in its language, and the box says, "Hello you are a human you are from planet Earth are you not." It seems that the box is a rudimentary translation device, speaking in a monotone voice and without pausing between sentences. It is somewhat jarring to listen to, but you understand what is being said.`
```